papaparse.js

```diff
@@ -28,7 +28,7 @@ License: MIT
 	// in strict mode we cannot access arguments.callee, so we need a named reference to
 	// stringify the factory method for the blob worker
 	// eslint-disable-next-line func-name
-}(this, function moduleFactory()
+}(typeof globalThis !== 'undefined' ? globalThis : this, function moduleFactory()
```

Comment:
Howdy @btakita and @pokoli!

Just wanted to bump this and see if it could get merged?
I could make the change to remove the ternary in favor of just `globalThis` if needed.